package.json

```diff
@@ -34,7 +34,7 @@
     "babel-preset-es2015": "^6.13.2",
     "chai": "^3.5.0",
     "coveralls": "~2.11.1",
-    "dockerode": "^2.4.3",
```

Comment:
Yeah maybe you can keep using new minor versions, or at least try again with `~2.4.3` before sticking to `2.4.3`.